### ISSUE TITLE
Custom Image in Bubble Icon Not Displaying in physical devices

### ISSUE THREAD

**Issue by dhiaCodes** [moazelsawaf/dash_bubble#18]
Hey there,
I'm encountering an issue when displaying a custom image within the bubble's icon attribute. When I set the `bubbleIcon` attribute to a custom image path, a black box appears instead of the image. However, the default icon provided by the package works perfectly.

```
 await BubuleHandler().startBubble(
            bubbleOptions: BubbleOptions(
              bubbleIcon: 'rounded',
            ),
```
### The icon is set as an image file placed inside:  `android\app\src\main\res\drawable\`
![Screenshot from 2024-05-07 12-33-25](https://github.com/moazelsawaf/dash_bubble/assets/84049068/c58ca4da-ea73-4969-8bc7-ab398095651c)

### Result on a physical device:
![Untitled design](https://github.com/moazelsawaf/dash_bubble/assets/84049068/fac41ad5-6fce-4e81-acfd-44cb4cc55a2e)

**PS: The custom image works fine on the virtual emulator.**
Please help identify the issue.

**Comment by dhiaCodes** [comment]
Still looking for a solution.

**Comment by tolgaakcaoglu** [comment]
I'm experiencing the same problem. It looks like the developer won't be helpful. I've been waiting for a solution to this issue for a while now.

**Comment by dhiaCodes** [comment]
@tolgaakcaoglu, For now, the only workaround I can do is to replace the initial package bubble image that can be found in `/home/$user/.pub-cache/hosted/pub.dev/dash_bubble-2.0.0/android/src/main/res/drawable` with the desired image with the same filename and format.
Then regenerate build assets by running `flutter pub get` in the project.
This is a temporary stupid solution that works perfectly.

**Comment by lehoangbaochung** [comment]
same issue on physical devices

**Comment by tolgaakcaoglu** [comment]
> @tolgaakcaoglu, For now, the only workaround I can do is to replace the initial package bubble image that can be found in `/home/$user/.pub-cache/hosted/pub.dev/dash_bubble-2.0.0/android/src/main/res/drawable` with the desired image with the same filename and format. Then regenerate build assets by running `flutter pub get` in the project. This is a temporary stupid solution that works perfectly.

You're right. I also solved my problem using this method. The package owner forgets to be helpful. The custom image error of the package needs to be fixed.

In addition, I would be very happy if they could do the following;

- Sending an image of type Uint8List on demand
